ksp_plugin/pile_up.cpp

```diff
@@ -416,6 +406,68 @@ void PileUp::AppendToPart(DiscreteTrajectory<Barycentric>::Iterator it) const {
   }
 }
 
+DegreesOfFreedom<Barycentric> PileUp::RecomputeFromParts(
+    std::list<not_null<Part*>> const& parts) {
+  // First compute the overall mass and centre of mass of the pile-up.  Also
+  // compute the overall force applied to it.
+  // TODO(phl): We assume that forces are applied at the centre of mass of the
+  // pile-up, but they are really applied at the centre of mass of the parts, so
+  // this introduces a torque.
```

Comment:
They are not even applied at the centre of mass of the parts; they are applied at some position, which we do not currently transmit to the C++.

Reply:
Fixed the comment.